src/AddItems.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import { db } from './lib/firebase';
+
+function AddItems(props) {
+  const [groceryItem, setGroceryItem] = useState('');
+  const [userToken, setUserToken] = useState('randomTokenOne');
+  const updateGroceryItem = (event) => {
+    setGroceryItem(event.target.value);
+  };
+  const submitGroceryItem = (event) => {
+    event.preventDefault();
+    const formData = {
+      itemName: groceryItem,
+      frequency: document.querySelectorAll('input[name=frequency]:checked')[0]
+        .value,
+      lastPurchaseDate: null,
+    };
+    db.collection(userToken).add({ formData });
+  };
+
+  return (
+    <div>
+      <form id="test-form" onSubmit={submitGroceryItem}>
+        <label htmlFor="item">Item name:</label>
+        <input
+          type="text"
+          id="item"
+          value={groceryItem}
+          onChange={updateGroceryItem}
+        />
+        <input type="radio" id="soon" value={7} name="frequency" />
+        <label htmlFor="soon">Soon (in the next 7 days)</label>
+        <input type="radio" id="kindOfSoon" value={14} name="frequency" />
+        <label htmlFor="kindOfSoon">Kind of soon (in the next 14 days)</label>
+        <input type="radio" id="notSoon" value={30} name="frequency" />
+        <label htmlFor="notSoon">Not soon (in the next 30 days)</label>
+        <input type="submit" value="Add item" />
```

Comment:
Currently, it's possible to submit the form without checking off a radio box or entering text, which causes an error above, `Cannot read property 'value' of undefined` since the radio box is empty. It would be a good idea to disable this input unless both the frequency and text inputs have a value.

Reply:
That makes sense. Is it better to disable the submit until all inputs are completed rather than making the inputs `required`? Or is it a good idea to do both?

Reply:
Oh yea you can do that too! Either way works 💯